### PR TITLE
Added form for new animals

### PR DESCRIPTION
**Pull request by Evirth**

### CODE REVIEW

**Comment by lsielewicz** [comment]
Shouldn't we add a new controller for this functionality?  I mean, all those changes are placed in HomeController. I think it should be changed.

**Comment by Astralius** [comment]
Doh... Definitely not in HomeController, but not in AddPet as well. If there is AddPetController, then you will need to add a separate RemovePetController and EditPetController soon... I would stick with the existing PetsController for all operations involving a single Pet. 
Please leave it to me: creating a controller for pet addition is a separate task.

Also... naming - we agreed to name these creatures Pets, not Animals. Please adjust the naming to be consistent with the rest of the code.

File: Toz.Dotnet/Toz.Dotnet/Views/AddPet/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<body>
+    <h2>@Localizer["Title"]</h2>
+    <hr>
+
+    <div class="AddingAnAnimal">
+        <form action="" method="POST">
+            <table id="AddAnimalTable" align="center">
+                <tr>
+                    <td><label for="AnimalName">@Localizer["Name"]:</label></td>
+                    <td><input type="text" name="AnimalName" id="AnimalName" maxlength="20"></td>
```
Comment (kkaniewski) [inline Toz.Dotnet/Toz.Dotnet/Views/AddPet/Index.cshtml]: Not sure if we need maxlength in here. We can validate length in our Model and it's a part of another task.
https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation

File: Toz.Dotnet/Toz.Dotnet/Views/AddPet/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                </tr>
+                <tr>
+                    <td><label for="AnimalType">@Localizer["Type"]:</label></td>
+                    <td>
+                        <select name="AnimalType" id="AnimalType">
+                            <option value="UnidentifiedType"></option>
```
Comment (kkaniewski) [inline Toz.Dotnet/Toz.Dotnet/Views/AddPet/Index.cshtml]: Not sure if we should add Unidentified option in our PetType enum or not.

File: Toz.Dotnet/Toz.Dotnet/Views/AddPet/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+<head>
+    <link rel="stylesheet" href="~/css/style.css" />
+</head>
+
+<body>
+    <h2>@Localizer["Title"]</h2>
+    <hr>
+
+    <div class="AddingAnAnimal">
```
Comment (kkaniewski) [inline Toz.Dotnet/Toz.Dotnet/Views/AddPet/Index.cshtml]: Pet not Animal. Related to whole file.

**Comment by Astralius** [comment]
Looking good, let's merge it in.

**Pull request merged**